src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -448,7 +445,7 @@ function getSecondaryReportActions(
 
     options.push(CONST.REPORT.SECONDARY_ACTIONS.VIEW_DETAILS);
 
-    if (isDeleteAction(report, reportTransactions, reportActions)) {
+    if (isDeleteAction(report, reportActions)) {
         options.push(CONST.REPORT.SECONDARY_ACTIONS.DELETE);
```

Comment:
I think we should update the `CONST.REPORT.SECONDARY_ACTIONS.DELETE` const to `deleteReport` so we can differentiate it from `CONST.REPORT.TRANSACTION_SECONDARY_ACTIONS.DELETE` and setup the proper action in `secondaryActionsImplementation`

Reply:
@allgandalf we're also missing the action setup [here](https://github.com/Expensify/App/blob/6977aaff24f420caa2d4718e5b6205acecc10cc3/src/components/MoneyReportHeader.tsx#L734-L741) (the current one is for the delete expense case) so we need to add one for the delete report case. And the correct modal/callback [here](https://github.com/Expensify/App/blob/6977aaff24f420caa2d4718e5b6205acecc10cc3/src/components/MoneyReportHeader.tsx#L972-L982)